src/components/AnchorForAttachmentsOnly/BaseAnchorForAttachmentsOnly.tsx

```diff
@@ -63,7 +63,7 @@ function BaseAnchorForAttachmentsOnly({style, source = '', displayName = '', dow
                     <AttachmentView
                         source={sourceURLWithAuth}
```

Comment:
Since we introduced local file for preview attachments, we also need to check here whether we should not add an encrypted token for the local file. 
More details: https://github.com/Expensify/App/issues/54640#issuecomment-2564648498